liquidator/liquidator.js

```diff
@@ -61,36 +65,35 @@ class Liquidator {
   update = async () => {
     await this.empClient.update();
     await this.gasEstimator.update();
+    await this.priceFeed.update();
   };
 
   // Queries underCollateralized positions and performs liquidations against any under collateralized positions.
-  queryAndLiquidate = async priceFunction => {
+  queryAndLiquidate = async () => {
     const { toBN, fromWei, toWei } = this.web3.utils;
+    const price = this.priceFeed.getCurrentPrice();
 
-    const contractTime = this.empClient.getLastUpdateTime();
-    const priceFeed = priceFunction(contractTime);
-
-    // The `priceFeed` is a Number that is used to determine if a position is liquidatable. The higher the
-    // `priceFeed` value, the more collateral that the position is required to have to be correctly collateralized.
-    // Therefore, we add a buffer by deriving a `scaledPriceFeed` from (`1 - crThreshold` * `priceFeed`)
-    const scaledPriceFeed = fromWei(toBN(priceFeed).mul(toBN(toWei("1")).sub(toBN(this.crThreshold))));
+    // The `price` is a BN that is used to determine if a position is liquidatable. The higher the
+    // `price` value, the more collateral that the position is required to have to be correctly collateralized.
+    // Therefore, we add a buffer by deriving a `scaledPrice` from (`1 - crThreshold` * `price`)
+    const scaledPrice = fromWei(price.mul(toBN(toWei("1")).sub(toBN(this.crThreshold))));
     this.logger.debug({
       at: "Liquidator",
       message: "Scaling down collateral threshold for liquidations",
-      scaledPriceFeed: scaledPriceFeed,
+      scaledPrice: scaledPrice,
       crThreshold: this.crThreshold
     });
 
     this.logger.debug({
       at: "Liquidator",
       message: "Checking for under collateralized positions",
-      inputPrice: priceFeed
+      inputPrice: price.toString()
```

Comment:
Ah this is something I missed. This should be `scaledPrice.toString()` since that's the price we're using now

Reply:
I see. Okay, will update to put both in the log.

Reply:
Done. PTAL.